src/main/java/net/dries007/tfc/common/blocks/rock/AqueductBlock.java

```diff
@@ -39,12 +38,13 @@
 import org.jetbrains.annotations.Nullable;
 
 
-public class AqueductBlock extends HorizontalDirectionalBlock implements IFluidLoggable
+public class AqueductBlock extends Block implements IFluidLoggable
```

Comment:
Please revert, this serves no purpose and is binary compat breaking.

Reply:
This was changed because the only 2 methods in HorizontalDirectionalBlock are being overwritten. And it was trying to override the methods from the HorizontalDirectionalBlock class rather than Block which was doing weird stuff with the deprecation warnings.